www/docs/en/dev/config_ref/index.md

```diff
@@ -189,7 +189,7 @@ platform. See [Customize icons topic](images.html) for more information.
     <allow-intent href="sms:*" />
 
 ## engine
-   Specifies what platform to restore during a prepare. For more details, see the [Platforms & Plugins Version Management](../platform_plugin_versioning_ref/index.html).
+   Specifies what platform to restore during a prepare. For more details, see the [Platforms & Plugins Version Management](../platform_plugin_versioning_ref/index.html#platform-versioning).
```

Comment:
"Manage platforms and plugins" doesn't really convey the intent behind the info included in that link. It brings to my mind actions like: add/rm/update platforms/plugins.

I think the original "Platforms & Plugins Versioning" was much clearer, versioning being the keyword here.